src/core/tests/test_app.py

```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
         self.assertEqual(self.app.main_window.app, self.app)
         self.assertActionPerformed(self.app.main_window, 'show')
 
+    def test_is_full_screen(self):
+        self.assertFalse(self.app.is_full_screen)
+
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+
+        self.app.set_full_screen(["window1", "window2", "window3"])
+        self.assertTrue(self.app.is_full_screen)
+
+        self.app.set_full_screen()
+        self.assertFalse(self.app.is_full_screen)
+        
```

Comment:
At column 1: (W293) blank line contains whitespace

Reply:
When you press enter to move to next line it created some tab spaces just delete it .